changelogs/fragments/update_requires_ansible.yaml

```diff
@@ -0,0 +1,3 @@
+---
+major_changes:
+  - Update Requires_ansible from 2.9.10 to 2.14.
```

Comment:
We should be more elaborate with this.
```
---
major_changes:
  - Bumping `requires_ansible` to `>=2.14.0`, since previous ansible-core versions are EoL now.
release_summary:
  - Starting from this release, the minimum `ansible-core` version this collection requires is `2.14.0`. That last known version compatible with ansible-core<2.14 is `v2.12.0`.
```